vizro-core/src/vizro/static/js/models/dashboard.js

```diff
@@ -1,3 +1,3 @@
 export function _update_dashboard_theme(on) {
-  return on ? "vizro_dark" : "vizro_light";
+  return on ? "vizro_light" : "vizro_dark";
```

Comment:
Maybe we should replace the "on" with the "checked" variable name even here.

Reply:
Makes sense 👍 When you are back - let's schedule a session where you can explain me these js tests 😄

Reply:
Would also like a refresher @petar-qb  If I recall correctly there are some js tests outstanding even no? for some clientside callback